### ISSUE TITLE
About the training details

### ISSUE THREAD

**Issue by xccyue** [y-zheng18/GIMO#4]
I'm currently working on a project involving this dataset, and I have a few questions regarding the training details. I would appreciate any insights or guidance you can provide.

1. Optimizer:
I would like to know what optimizer was used for training and the settings of the optimizer.

2. Loss:
Could you please clarify the choice of loss function used during training? I saw "lambda_rec" and "lambda_des" in the config.py.

3. Pretrained Scene Encoder:
I am not sure whether this project uses pretrained scene encoder.

And it would be much appreciated if I could have the training code or more details of the training process.

**Comment by y-zheng18** [comment]
You can use Adam as optimizer, with 1e-4 lr and 5e-4 weight decay (set in config.py). Both reconstruction loss and destination loss are used, where reconstruction loss is used to supervise the prediction of the 5s sequence, and destination loss aims to supervise the prediction of the last pose of the whole sequence. The scene encoder is trained from scratch.

**Comment by xccyue** [comment]
Thanks so much for your help!

**Comment by Jackiemin233** [comment]
Thank you for your great work. Could you please provide the details about training batchsize and the scene pointcloud numbers?
